test/wrappers/ortools_test.rb

```diff
@@ -5770,4 +5770,27 @@ def test_self_selection_should_not_change_vehicle_ids
       end
     end
   end
+
+  def test_quantity_precision
+    problem = VRP.basic
+    problem[:services].each{ |service|
+      service[:quantities] = [{ unit_id: 'kg', value: 1.001 }]
+    }
+    problem[:vehicles].each{ |vehicle|
+      vehicle[:capacities] = [{ unit_id: 'kg', limit: 3 }]
+    }
+
+    result = OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:ortools] }}, TestHelper.create(problem), nil)
+    assert_equal 2, result[:routes].first[:activities].count{ |activity| activity[:service_id] },
+                 'The route is expected to contain 2 services'
```

Comment:
I think there is an indentation problem here

Reply:
the second line is aligned with the first parameter which is `2`